ts/features/itwallet/common/components/ItwSkeumorphicCard/CardClaim.tsx

```diff
@@ -71,6 +73,15 @@ const CardClaim = ({
         claim?.value,
         ClaimValue.decode,
         E.fold(constNull, decoded => {
+          if (
+            NestedArrayClaim.is(decoded) ||
+            DrivingPrivilegesValueRaw.is(decoded)
```

Comment:
`DrivingPrivilegesValueRaw.is(decoded)` should be placed in the `DrivingPrivilegesClaim.is(decoded)` branch. What do you think?